easybuild/easyblocks/b/bwise.py

```diff
@@ -0,0 +1,107 @@
+##
+# Copyright 2018 Free University of Brussels
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/easybuilders/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+@author: Ward Poelmans (Free University of Brussels)
+"""
+import glob
+import os
+import stat
+
+from easybuild.easyblocks.generic.makecp import MakeCp
+from easybuild.framework.easyconfig import CUSTOM
+from easybuild.tools.build_log import EasyBuildError
+from easybuild.tools.filetools import adjust_permissions, apply_regex_substitutions, copy_file, write_file
+
+
+class EB_BWISE(MakeCp):
+    """
+    Custom easyblock to install BWISE
+    """
+    @staticmethod
+    def extra_options(extra_vars=None):
+        """Change default values of options"""
+        extra = MakeCp.extra_options()
+        # files_to_copy is not mandatory here
+        extra['files_to_copy'][2] = CUSTOM
+        return extra
+
+    def build_step(self, verbose=False, path=None):
+        """Run multiple times for different sources"""
+
+        makefiles_fixes = [
+            (r'^CC=.*$', 'CC=$(CXX)'),
+            (r'^CFLAGS=.*$', 'CFLAGS:=$(CFLAGS)'),
+            (r'^LDFLAGS=.*$', 'LDFLAGS:=$(LDFLAGS) -fopenmp')
```

Comment:
Hmm, maybe it's better to set the `openmp` toolchain option to `True` rather than hardcoding this? Hardcoding `-fopenmp` will work for GCC & (recent enough) Intel compilers, but not for others...

Reply:
The issue is that `-fopenmp` is not adding to `LDFLAGS`, only `CFLAGS`